js1-2-8.js

```diff
@@ -12,8 +12,8 @@ if(typeof(firstNumber) == typeof(secondNumber)){
 /* let firstNumber = 7,
 secondNumber = 10; */
 
-let firstString = "10",
-    first = 0;
+let firstString = ture,
```

Comment:
this is not valid a boolean value (and will not compile). If you can fix this then it is fine